AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -254,6 +254,15 @@ - (void)setFrame:(CGRect)rect
       CGPoint newPosition = CGPointZero;
       ASBoundsAndPositionForFrame(rect, origin, anchorPoint, &newBounds, &newPosition);
 
+      if (ASIsCGRectValidForLayout(newBounds) == NO) {
+        ASDisplayNodeAssert(NO, @"-[ASDisplayNode setFrame:] - The new bounds are invalid and unsafe to be set.");
```

Comment:
This is a great change, thank you @rmls!!  Could you make a small modification by including the values being attempted to set?

I think you could do this like: ... setFrame:%@ ...", NSStringFromCGRect(newBounds)